dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/IdentifiableObjectStore.java

```diff
@@ -430,26 +391,26 @@ T getByUniqueAttributeValue(
    * Look up list objects which have property createdBy or lastUpdatedBy linked to given {@link
    * User}
    *
-   * @param user the {@link User} for filtering
+   * @param userDetails the {@link User} for filtering
```

Comment:
javadoc @link param should reflect the change to use the `UserDetails` class also, as that is what will be used for filtering not the `User` class.